live/src/main/java/org/dbpedia/extraction/live/administrative/AddItemServlet.java

```diff
@@ -0,0 +1,44 @@
+package org.dbpedia.extraction.live.administrative;
+
+import org.dbpedia.extraction.live.queue.LiveQueue;
+import org.dbpedia.extraction.live.queue.LiveQueueItem;
+import org.dbpedia.extraction.live.queue.LiveQueuePriority;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Created by Andre on 31/07/2015.
+ */
+public class AddItemServlet extends HttpServlet {
+    public AddItemServlet(){}
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException
+    {
+        response.setContentType("text/html");
```

Comment:
I think it would be better to use `HttpServletResponse.SC_FORBIDDEN` [1] in case of a bad password
and the following lines are duplicated along all similar servlets. Better encapsulate the logic in a function or inheritance

[1] https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServletResponse.html#SC_FORBIDDEN